ReactAndroid/src/main/java/com/facebook/react/devsupport/DevSupportManagerImpl.java

```diff
@@ -119,6 +122,8 @@
   private @Nullable RedBoxDialog mRedBoxDialog;
   private @Nullable AlertDialog mDevOptionsDialog;
   private @Nullable DebugOverlayController mDebugOverlayController;
+  private @Nullable DevLoadingView mDevLoadingView;
+  private boolean mDevLoadingViewVisible = false;
```

Comment:
Do you need this flag? How about just checking the visibility/existence of mDevLoadingView to determine if it's visible?